tools/cvm-image-rewriter/plugins/07-device-permission/README.md

```diff
@@ -0,0 +1,3 @@
+# Device permission
+
+This plugin is used to setup udev rules for CCNP device plugin. Users do not need to customize.
```

Comment:
Why make it a plugin then?  Why not just set this as part of the main install script?  Unless you think people want to customize udev rules in the future I would remove this option.  You can paralyze users by choice if you give too many options.

Reply:
In fact, the plugin form is more convenient for flexible operations and maintenance. Users can customize udev rules in the future as TDX is updated or device-plugin is updated, but currently, we do not recommend doing so.

Reply:
I think then I am confused by the  "Users do not need to customize"   Maybe it would be better to say   "It is typically not required for users to customize unless additional udev rules are needed."